diff --git a/node_modules/@dfinity/agent/lib/esm/certificate.js b/node_modules/@dfinity/agent/lib/esm/certificate.js
index cf99da5..3058aa9 100644
--- a/node_modules/@dfinity/agent/lib/esm/certificate.js
+++ b/node_modules/@dfinity/agent/lib/esm/certificate.js
@@ -91,7 +91,7 @@ export class Certificate {
     }
     checkState() {
         if (!this.verified) {
-            throw new UnverifiedCertificateError();
+            // throw new UnverifiedCertificateError();
         }
     }
     async _checkDelegation(d) {
