diff --git a/node_modules/iso-url/src/url.js b/node_modules/iso-url/src/url.js
index a57894b..7d08c83 100644
--- a/node_modules/iso-url/src/url.js
+++ b/node_modules/iso-url/src/url.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const { URL, URLSearchParams, format } = require('url');
+const { format } = require('url');
 
 // https://github.com/nodejs/node/issues/12682
 const defaultBase = 'http://localhost';
